eras/alonzo/impl/src/Cardano/Ledger/Alonzo/PParams.hs

```diff
@@ -484,7 +485,7 @@ instance FromJSON (AlonzoPParams Identity era) where
         <*> obj .: "decentralization"
         <*> obj .: "extraPraosEntropy"
         <*> obj .: "protocolVersion"
-        <*> obj .: "minPoolCost" .!= mempty
+        <*> obj .:? "minPoolCost" .!= mempty
```

Comment:
There is really no such thing as an optional protocol parameter. I think that was just a mistake from a long time ago.
```suggestion
        <*> obj .: "minPoolCost"
```

Reply:
Ok! 
I also removed the optionality for shelly in a separate commit, for `minUTxOValue` and `minPoolCost`